homeassistant/components/motioneye/camera.py

```diff
@@ -86,7 +86,7 @@ def camera_add(camera: dict[str, Any]) -> None:
     listen_for_new_cameras(hass, entry, camera_add)
 
 
-class MotionEyeMjpegCamera(MjpegCamera, CoordinatorEntity):
+class MotionEyeMjpegCamera(MjpegCamera, CoordinatorEntity[Optional[Dict[str, Any]]]):
```

Comment:
Can we not use new style optional (PEP604) and collections (PEP585) typing?

Reply:
In generics we have to use old types.